NewHaven/src/Player.h

```diff
@@ -34,7 +34,7 @@ struct Player
     void printHarvestCards();
     void printBuildingCards();
     int calculateScore() const;
-    VGMap getVillage() const;
+    VGMap& getVillage() const;
     Hand getHand() const;
```

Comment:
yes and no, making village public would allow them to alter the pointer directly and redirect to another village or even delete it. I think returning it this way is good, but might not bed necessary since me dumb and then thing of resetting the states after the notify >_< we can just pass a copy which would be ideal